lib/addons/flagger/index.ts

```diff
@@ -0,0 +1,72 @@
+import 'source-map-support/register';
```

Comment:
we don't need this import. Why was it added?

Reply:
Not sure might have been from a quick fix I did but it was not faded so I assumed it was doing something. I have deleted it and see it did not mess anything up.